packages/react-charting/webpack.config.js

```diff
@@ -1,9 +1,39 @@
 const { resources } = require('@fluentui/scripts-webpack');
+const TerserPlugin = require('terser-webpack-plugin');
 
 module.exports = [
   // Create a bundle for consumption in the browser
   ...resources.createBundleConfig({
     output: 'FluentUIReactCharting',
+    customConfig: {
+      optimization: {
+        minimizer: [
+          new TerserPlugin({
+            parallel: true,
+            terserOptions: {
+              // https://github.com/terser/terser
+              mangle: {
+                // properties: {
+                //   regex: /^_/,
+                // },
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                keep_classnames: false,
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                keep_fnames: false,
+                properties: {
+                  //regex: /^_/,
+                  regex: /((?<=\b|\.)(?=\b[a-zA-Z$_])\w+)|^_/g,
+                },
+              },
+              output: {
+                beautify: false,
```

Comment:
beautify option is false by default.
https://terser.org/docs/options/#format-options